README.md

```diff
@@ -1,24 +1,29 @@
+<h1 align="center"> GitPoint </h1> <br>
 <p align="center">
   <a href="https://gitpoint.co/">
     <img alt="GitPoint" title="GitPoint" src="http://i.imgur.com/VShxJHs.png" width="450">
   </a>
 </p>
 
 <p align="center">
-  GitHub for iOS. Built with React Native.  </p>
+  GitHub for iOS. <a href="https://github.com/gitpoint/git-point/issues/2#issuecomment-316261018">Android comming soon </a> <br>
+</p>
+
+<p align="center">
+  <a href="
+https://travis-ci.org/gitpoint/git-point"><img src="https://img.shields.io/travis/gitpoint/git-point.svg?style=flat-square" alt="Build Status"></a>
+  <a href="#contributors"><img src="https://img.shields.io/badge/all_contributors-18-orange.svg?style=flat-square" alt="All Contributors"></a>
```

Comment:
Personally I think everything should be out of HTML (except things that really need it like center tags). All images and link should be markdown for consistency and readability's sake.